alacritty/src/renderer/mod.rs

```diff
@@ -441,6 +458,15 @@ impl Batch {
             self.tex = glyph.tex_id;
         }
 
+        let mut cell_flags = RenderingGlyphFlags::empty();
+        if glyph.multicolor != 0 {
+            cell_flags |= RenderingGlyphFlags::COLORED;
+        }
+
+        if cell.flags.contains(Flags::WIDE_CHAR) {
+            cell_flags |= RenderingGlyphFlags::WIDE_CHAR;
+        }
```

Comment:
Can we use the methods provided by bitflags? They should be easier to understand and thus maintain.

Even better if we just do `cell_flags.set(COLORED, glyph.multicolor != 0)`. Then it's also shorter!